src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/WorkBasketInputDetails.scala

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.util.{CcdTokenGenerator, PerformanceTestsConfig}
+
+import scala.concurrent.duration._
+
+object WorkBasketInputDetails extends PerformanceTestsConfig {
+
+  private val url: String = config.getString("caseDataUrl") + "/" + config.getString("workbasketInputDetails")
+  println("Retrieving workbasketInputDetails URL : " + url)
+
+  def httpRequest() = {
+    val s2sToken = CcdTokenGenerator.generateGatewayS2SToken()
+    val userToken = CcdTokenGenerator.generateWebUserToken()
+    http("TX08_CCD_WorkBasketInputDetails_workbasketInputDetails")
+      .get(url)
+      .header("ServiceAuthorization", s2sToken)
+      .header("Authorization", userToken)
+      .header("Content-Type","application/json")
+      .check(status in  (200))
```

Comment:
remove not needed spaces please

Reply:
not clear about this. remove what ?

Amended println text and reformatted code.